### PR TITLE
Update jq version checking logic and fixed some typos.

### PR DESCRIPTION
**Pull request by KaiGuo-gkai**
- We have a check to see if the user has _jq_ installed. But check logic does the opposite. The PR fixed the logic.
- We have a Python script written in Python 3. An error occurs when the default local version is Python 2. Specifying the Python version solves this problem. (Another solution would be to modify the script so that both versions work)
- Fixed some typos.

### CODE REVIEW

File: build/README.md [hunk 1]
```diff
@@ -171,12 +171,12 @@ a PR to that effect would be greatly appreciated.
 1.  Use [`make-certs.py` script](./make-certs.py) to create certificates for
     the new CockroachDB cluster:
 
-        ./make-certs.py --cluster $CLUSTER_CONTEXT --namespace $NAMESPACE
+        python3 ./make-certs.py --cluster $CLUSTCONTEXT --namespace $NAMESPACE
```
Comment (BenjaminPelletier) [inline build/README.md:174]: I think we want to keep `$CLUSTER_CONTEXT` rather than replacing with `$CLUSTCONTEXT`

Comment (BenjaminPelletier) [inline build/README.md:174]: To be consistent with other script usage in this repository, could we instead change the first line of `make-certs.py` to be the following?

```python
#!/usr/bin/env python3
```

**Review by BenjaminPelletier — COMMENTED** [review]
Looks good -- just 1 change request and 1 suggestion

File: build/README.md (hunk 1 above)
Comment (KaiGuo-gkai) [reply build/README.md:174]: Oops, it is a typo. Thank you for pointing out.

Comment (KaiGuo-gkai) [reply build/README.md:174]: Good idea. Will do.

**Review by BenjaminPelletier — APPROVED** [review]

**Pull request merged**